openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -82,6 +82,12 @@ <h3 class="hd-6">${_("Course Tools")}</h3>
                                     ${_("Updates")}
                                 </a>
                             </li>
+                            <li>
```

Comment:
I think we'd decided that this would be conditional based on a Waffle flag, right?